src/sentry/types/issues.py

```diff
@@ -50,7 +50,7 @@ class GroupCategory(Enum):
     GroupType.PERFORMANCE_FILE_IO_MAIN_THREAD: "File IO on Main Thread",
     GroupType.PERFORMANCE_N_PLUS_ONE_API_CALLS: "N+1 API Calls",
     GroupType.PERFORMANCE_M_N_PLUS_ONE_DB_QUERIES: "MN+1 Query",
-    GroupType.PERFORMANCE_UNCOMPRESSED_ASSETS: "Uncompressed Assets",
+    GroupType.PERFORMANCE_UNCOMPRESSED_ASSETS: "Uncompressed Asset",
```

Comment:
Should we rename it to `GroupType.PERFORMANCE_UNCOMPRESSED_ASSET` as well, or is that gonna be a pain?